rest/model/task_annotations.go

```diff
@@ -155,3 +160,27 @@ func ArrAPIIssueLinkArrtaskannotationsIssueLink(t []APIIssueLink) []annotations.
 	}
 	return m
 }
+
+func GetJiraTickets(issuelinks []APIIssueLink) ([]APIIssueLink, error) {
+	settings := evergreen.GetEnvironment().Settings()
+	jiraHandler := thirdparty.NewJiraHandler(*settings.Jira.Export())
+	catcher := grip.NewBasicCatcher()
+
+	var res []APIIssueLink
+	for _, issue := range issuelinks {
+		urlObject, err := url.Parse(*issue.URL)
+		catcher.Wrap(err, "problem parsing the issue url")
+		if urlObject == nil {
```

Comment:
Sorry, this isn't exactly what I meant; if urlObject equals nil, we just want to skip 176 -182, but we still want to hit 183 (i.e. we don't want to leave this issue out completely because the URL is malformed). Can you just modify the if statement at 176?

Reply:
fixed